paper/paper.tex

```diff
@@ -0,0 +1,272 @@
+%                 \item Standard error can be thought of as the variation if the sampling was done repeated. 
+%             \end{itemize}
+%         \item Usage:
+%             \begin{enumerate}
+%                 \item Generate replicate weights using bootstrap, jackknife, BRR, etc. 
+%                 \item Using each replicate weight, calculate the estimate. 
+%                 \item Calculate the standard error using the new set of estimates. 
+%             \end{enumerate}
+%         \end{itemize}
+
+\subsection{Bootstrapping}
+
+
+
+In the bootstrap method, each replicate \( r \) involves selecting a simple random sample of \( n_h - 1 \) primary sampling units (PSUs) with replacement from the \( n_h \) sample PSUs in stratum \( h \). The adjusted weight \( w_i'(r) \) for observation \( i \) in replicate \( r \) is calculated as:
+
+% For bootstrap replicate $r (r = 1, \dots, R)$, an SRS of $n_h - 1$ PSUs is selected with replacement from the $n_h$ sample PSUs in stratum $h$. $m_{hj}(r)$ represents the number of times PSU $j$ of stratum $h$ is selected in replicate $r$.
+
+% The adjusted weight $w_i'(r)$ for observation $i$ in replicate $r$ is calculated as:
+
+\begin{equation}
+    w_i'(r) = w_i(r) \times \frac{n_h}{n_h - 1} \times m_{h}(r)
+\end{equation}
+
+Where \( w_i(r) \) denotes the initial weight for observation \( i \) within replicate \( r \), \( n_h \) is the total number of observations in stratum \( h \), and \( m_{h}(r) \) is the number of PSUs in stratum \( h \) that are selected in replicate \( r \)\cite{Lohr}.
+
+\verb|bootweights| can be used to generate \verb|ReplicateDesign{BootstrapReplicates}| from a \verb|SurveyDesign|. 
+
+\begin{lstlisting}
+julia> bdesign = bootweights(design; replicates = 1000)
+\end{lstlisting}
+
+
+% \begin{lstlisting}
+% julia> srs = SurveyDesign(apisrs; weights=:pw);
+
+% julia> bsrs = bootweights(srs; replicates = 1000)
+% ReplicateDesign{BootstrapReplicates}:
+% data: 200x1045 DataFrame
+% strata: none
+% cluster: none
+% popsize: [6194.0, 6194.0, 6194.0  ...  6194.0]
+% sampsize: [200, 200, 200  ...  200]
+% weights: [30.97, 30.97, 30.97  ...  30.97]
+% allprobs: [0.0323, 0.0323, 0.0323  ...  0.0323]
+% type: bootstrap
+% replicates: 1000
+% \end{lstlisting}
+
+The replicate design object facilitates variance estimation. When a function receives a \verb|ReplicateDesign| rather than a \verb|SurveyDesign|, it provides the standard error along with the point estimate.
+For example: 
+\begin{lstlisting}
+julia> mean(:api99, bdesign)
+1x2 DataFrame
+    Row | mean     SE      
+        | Float64  Float64 
+--------|-----------------
+      1 | 606.978  24.7505
+    \end{lstlisting}
+For each replicate $r$, $\hat{\theta}^*_r$ is the estimator of $\theta$, calculated the same way as $\hat{\theta}$ but using weights $w_i'(r)$ instead of the original weights $w_i$. The variance of the estimator is given by: 
```

Comment:
```suggestion
For each replicate $r$, $\hat{\theta}^*_r$ is calculated using the adjusted weights $w_i'(r)$. $\hat{\theta}^*$ instead of the original weights $w_i$. The variance of the estimator is estimated as: 
```